tests/Backend/Mixed/SharingTest.php

```diff
@@ -345,6 +353,25 @@ public function testSharedBuckets($backend)
             $this->assertEquals($sharedBucket['project']['id'], $project['id']);
             $this->assertEquals($sharedBucket['project']['name'], $project['name']);
 
+            $this->assertArrayHasKey('id', $sharedBucket['sharedBy']);
```

Comment:
Este by som tu otestoval ci tam je to `sharingParameters` ktore si tam pridal v connection